internal/summary/summary.go

```diff
@@ -70,47 +75,38 @@ func (s *Summary) Gather(services []model.Service) error {
 }
 
 func (s *Summary) Output() string {
-	ret := "\n──┤ Services Built … ├────────────────────────────────────────\n"
-
+	ret := ""
 	for _, svc := range s.Services {
-		ret += fmt.Sprintf("    SERVICE NAME: %s\n", svc.Name)
-		ret += fmt.Sprintf("    WATCHING: %s\n", svc.Path)
-
-		k := svc.k8sData
+		indent := " "
+		ret += fmt.Sprintf("%s%s — ", indent, svc.Name)
 
-		ret += fmt.Sprintln("    KUBERNETES INFO")
-		if len(k.Version) > 0 {
-			ret += fmt.Sprintf("      • Version: %s\n", k.Version)
+		// Relative Path
+		wd, err := os.Getwd()
+		if err != nil {
+			log.Fatalf("Failed to get working directory: %s", err)
```

Comment:
Not sure if I should toss away these errors instead? Is this too tedious?

Reply:
if we can't get the working director something is VERY WRONG, this is fine